scss/partials/fieldsets/_datesFieldset.scss

```diff
@@ -0,0 +1,57 @@
+input[type="date"] {
+    padding: 4px 5px;
+
+    &::-webkit-inner-spin-button {
+        display: none;
+    }
+
+    &::-webkit-calendar-picker-indicator {
+        @include defaultTag;
+        right: 1%;
+        z-index: 2;
+        opacity: 0;
+        cursor: pointer;
+    }
+}
+
+select {
+    @include default;
+    @include defaultBorder;
+}
+
+textarea {
+    @include default;
+    @include defaultBorder;
+    width: calc(100% - 12px);
+    padding: 5px;
+}
+
+.item {
+    &:hover {
+        p {
+            color: $green;
+        }
+        i {
+            color: $green;
+        }
+    }
+
+    select {
+        &:hover {
+            @include defaultHover;
+        }
+    }
+
+    textarea {
+        &:hover {
+            @include defaultHover;
+        }
+    }
+
+    i {
+        @include defaultTag;
+        right: 1%;
+        top: 30px;
+        z-index: 1;
+    }
+}
```

Comment:
This css in file only applied for Reservation Form, you need to put it into a file like _reservationForm.scss, not common filedset.
Note the you should use nested to make it more structure, please check the exercise again and think how to make structure for your project.